src/ec/packages/ec-component-text-area/adapter.js

```diff
@@ -13,10 +13,14 @@ const adapter = initialData => {
     delete adaptedData.is_disabled;
   }
 
-  adaptedData.helper_text = "This is the input's helper text";
+  adaptedData.helper_text = "This is the input's helper text.";
   adaptedData.optional_text = '(optional)';
-  adaptedData.name = 'example-name';
   adaptedData.label = 'Label';
+  adaptedData.required = true;
+  adaptedData.required_text = '*';
+  adaptedData.optional_text = '(optional)';
+  adaptedData.id = 'example';
+  adaptedData.name = '';
```

Comment:
We don't really need to add the key if we are not setting any value, this is not set in the specs so there's nothing to override.